pkg/services/govmomi/ipam/status_test.go

```diff
@@ -49,7 +49,7 @@ func Test_buildIPAMDeviceConfigs(t *testing.T) {
 	)
 
 	before := func() {
-		vmCtx = *fake.NewVMContext(fake.NewControllerContext(fake.NewControllerManagerContext()))
+		vmCtx = *fake.NewVMContext(ctx, fake.NewControllerContext(fake.NewControllerManagerContext()))
 		ctx = context.Background()
```

Comment:
We have to swap them, otherwise we have an empty context.
```suggestion
		ctx = context.Background()
		vmCtx = *fake.NewVMContext(ctx, fake.NewControllerContext(fake.NewControllerManagerContext()))
```